### ISSUE TITLE
Vestibule-ocular reflex abnormality

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#3583]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label: 
Vestibule-ocular reflex abnormality

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
The inability to stabilize images onto the retina during head movements.  

@psergouniotis would you agree? Is this term still in use?

### 4. Parent term 
Abnormal involuntary eye movements HP:0012547

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Multiple sclerosis.  MONDO'0005301
Annotating PMID: 11309470

**Comment by pnrobinson** [comment]
This term already exists:
https://hpo.jax.org/app/browse/term/HP:0007670
Note that "Vestibule-ocular reflex abnormality" is not a correct spelling in English (the "e" in "Vestibule" should be an "o")
